ansible/vagrant-workers.yml

```diff
@@ -44,74 +39,54 @@
           - zlib1g-dev
       become: yes
 
-    - name: Download girder
-      git:
-        repo: "https://github.com/girder/girder.git"
-        dest: "{{ girder_path }}"
-        version: "{{ girder_version }}"
-        update: yes
-        force: yes
-      register: result
-
-    - name: Install Python package
-      pip:
-        name: "{{ girder_path }}"
-        #    editable: true
-        # TODO: The 'editable' option is broken until Ansible 2.4
-        # See: https://github.com/ansible/ansible/pull/19688
-        extra_args: '-e'
-        virtualenv: "{{ python_dist_path }}"
-      when: result.changed
-
-    - name: Download isic-archive
-      git:
-        repo: "https://github.com/ImageMarkup/isic-archive.git"
-        dest: "{{ isic_archive_path }}"
-        version: "{{ isic_archive_version }}"
-        clone: "{{ isic_archive_clone }}"
-        update: no
-        force: yes
-
     - name: Install package dependencies
       apt:
         name:
           - p7zip-full
       become: yes
 
-    - name: Install isic-archive-tasks package
+    - name: Install isic-archive package
+      pip:
+        name: "{{ isic_archive_path }}"
+        virtualenv: "{{ python_dist_path }}"
+        editable: true
+
+    # TODO: Remove when girder-worker stops depending on networkx
+    - name: Install networkx==1.9.0
       pip:
-        name: "{{ isic_archive_path }}/isic-archive-tasks"
+        name: "networkx"
+        version: "1.9.0"
         virtualenv: "{{ python_dist_path }}"
 
+    # TODO: Remove when https://github.com/girder/girder_worker/issues/330 is resolved
+    - name: Install celery==4.1.1
+      pip:
+        name: "celery"
+        version: "4.1.1"
+        virtualenv: "{{ python_dist_path }}"
 
     - name: Configure mongo URI
       ini_file:
-        path: "{{ girder_path }}/girder/conf/girder.local.cfg"
+        path: "{{ ansible_user_dir }}/.girder/girder.cfg"
         section: database
         option: uri
         value: '"mongodb://{{ webserver_ip }}/girder"'
 
   roles:
-    - role: celery
-      archive_api_url: "http://isic-archive.test/api/v1"
-      celery_bin: "{{ python_dist_path }}/bin/celery"
-      celery_concurrency: 1
-      celery_user: vagrant
-
     - role: large_image
+      large_image_version: "girder-3"
       large_image_virtualenv: "{{ python_dist_path }}"
       large_image_tile_sources:
         - tiff
       large_image_include_vips: true
 
-  post_tasks:
-    - name: Install large_image plugin
-      command: "{{ python_dist_path }}/bin/girder-install plugin --force --symlink {{ large_image_path }}"
-
-    - name: Install ISIC Archive
-      command: "{{ python_dist_path }}/bin/girder-install plugin --force --symlink {{ isic_archive_path }}"
-      notify: Reload Celery
+    - role: celery
```

Comment:
If there's a reason that the `celery` role needs to happen after `large_image`, can you add a code comment explaining it?